### PR TITLE
optim logprobs

### PR DESCRIPTION
**Pull request by pathfinder-pf**
## Motivation
This optimization includes only returning the output log probabilities.

## Tests

launch server
- JAX_COMPILATION_CACHE_DIR=/tmp/jit_cache python3 -u -m sgl_jax.launch_server --model-path deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B --trust-remote-code --dist-init-addr=0.0.0.0:10011 --nnodes=1 --tp-size=1 --device=tpu --random-seed=27 --node-rank=0 --mem-fraction-static=0.8 --chunked-prefill-size=8192 --download-dir=/tmp --dtype=bfloat16 --precompile-bs-paddings 1 64 --max-running-requests 64 --max-total-tokens 257536 --skip-server-warmup --attention-backend=fa --precompile-token-paddings 8192 --page-size=64 --disable-overlap-schedule --log-requests --log-requests-level=3 --enable-precision-tracer --use-sort-for-toppk-minp
- eval
 curl localhost:30000/v1/chat/completions \
  -H "Content-Type: application/json" \
  -d '{
    "model": "qwen",
    "messages": [
      {
        "role": "user",
        "content": "haha hi~ please introduce yourself"
      }
    ],
    "temperature": 0.7,
    "logprobs": true,
    "max_tokens": 10
  }'
- result
image

- eval 
curl localhost:30000/v1/chat/completions   -H "Content-Type: application/json"   -d '{
    "model": "qwen",
    "messages": [
      {
        "role": "user",
        "content": "haha hi~ please introduce yourself"
      }
    ],
    "temperature": 0.7,
    "max_tokens": 10
  }'
image

## Checklist

- [x] Please use English, otherwise it will be closed.
- [x] The purpose of the PR, or link existing issues this PR will resolve.
- [x] The test plan, such as providing test command.
- [x] (Optional) The necessary documentation update.

### CODE REVIEW

File: python/sgl_jax/srt/managers/schedule_batch.py [hunk 1]
```diff
@@ -581,7 +584,7 @@ def init_new(
         mesh: mesh_lib.Mesh = None,
     ):
         return_logprob = any(req.return_logprob for req in reqs)
-
+        return_output_logprob_only = any(req.return_output_logprob_only for req in reqs)
```
Comment (aolemila) [inline python/sgl_jax/srt/managers/schedule_batch.py]: any -> all?

If used any, requests who require other logprobs would get wrong result

Comment (pathfinder-pf) [reply python/sgl_jax/srt/managers/schedule_batch.py]: Adjusted
